Cargo.toml

```diff
@@ -126,6 +126,12 @@ features = ["std"]
 [dependencies.percent-encoding]
 version = "2.1"
 
+[dependencies.moka]
+git = "https://github.com/moka-rs/moka"
```

Comment:
Note that you can't publish to crates.io with git dependencies, so this dependency needs to be changed to a crates.io dependency before 0.11 release

Reply:
Yeah, I know. I'm waiting for 0.8.0 on moka.